### PR TITLE
[AC-659] Add Contact Us Screen.

### PR DESCRIPTION
**Pull request by ribhavsharma**
## Description of what I changed

Added a Contact Us Screen that sends an email to the OpenMRS HelpDesk.  
- [x] Added it in the overflow menu.
- [x] Added it in the settings fragment.

## Screenshot
Screenshot_20200117-220611

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-659

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    MaterialButton send;
+    TextInputEditText name, subject, message;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_contact_us);
+        initViewFields();
+        send.setOnClickListener(v -> {
+            String name_str = name.getText().toString();
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: These 3 variables should be named using `camelCase`.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+public class ContactUsActivity extends AppCompatActivity {
+    MaterialButton send;
+    TextInputEditText name, subject, message;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_contact_us);
+        initViewFields();
+        send.setOnClickListener(v -> {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]:
```suggestion
        send.setOnClickListener(view -> {
```

or `_` if you're not using it

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import com.google.android.material.button.MaterialButton;
+import com.google.android.material.textfield.TextInputEditText;
+import org.openmrs.mobile.R;
+
+
+public class ContactUsActivity extends AppCompatActivity {
+    MaterialButton send;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]:
```suggestion
    MaterialButton sendButton;
```

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    TextInputEditText name, subject, message;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: Append `TIET` to each variable name. Also, can't these be private?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            String name_str = name.getText().toString();
+            String subject_str = subject.getText().toString();
+            String msg_str = message.getText().toString();
+
+            Intent sendMail = new Intent(Intent.ACTION_SEND);
+            sendMail.putExtra(Intent.EXTRA_EMAIL, new String[]{"helpdesk@openmrs.com"});
+            sendMail.putExtra(Intent.EXTRA_SUBJECT, name_str + " wants to get in touch.");
+            sendMail.putExtra(Intent.EXTRA_TEXT, "Regarding: " + subject_str + "\n" + msg_str);
+            sendMail.setType("message/rfc822");
+            startActivity(Intent.createChooser(sendMail, "Choose an Email Client"));
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: The string should be in `strings.xml` file.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java [hunk 6]
```diff
@@ -45,6 +48,7 @@
         boolean isDarkModeActivated();
 
         void setDarkMode(boolean darkMode);
+
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java]: This isn't necessary.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java [hunk 7]
```diff
@@ -34,6 +34,9 @@
         void rateUs();
 
         void setDarkMode();
+
+        void contactUs();
+
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java]: Just delete this line 39 for consistency.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java [hunk 8]
```diff
@@ -191,6 +193,15 @@ public void setDarkMode() {
         });
     }
 
+    @Override
+    public void contactUs() {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java]: Rename to `setUpContactUsPage`

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java [hunk 9]
```diff
@@ -14,7 +14,12 @@
 
 package org.openmrs.mobile.activities.settings;
 
+import android.content.Intent;
+import android.view.View;
+import android.widget.LinearLayout;
+import org.openmrs.mobile.R;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java]: Are these imports needed?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java [hunk 10]
```diff
@@ -94,4 +99,6 @@ public boolean isDarkModeActivated() {
     public void setDarkMode(boolean darkMode) {
         ThemeUtils.setDarkMode(darkMode);
     }
+
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java]: Remove line 102 and 104.

File: openmrs-client/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -13,346 +13,354 @@
   -->
 
 <resources>
-    <string name="app_name">OpenMRS Client</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]: Have you set up your IDE indents properly? See [our code style](https://github.com/openmrs/openmrs-contrib-android-client#Code-Style) section.

File: openmrs-client/src/main/res/layout/activity_contact_us.xml [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+    -->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	xmlns:tools="http://schemas.android.com/tools"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:background="#FFF">
+
+	<TextView
+		android:id="@+id/contact_tv"
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/layout/activity_contact_us.xml]: Inconsistent indentation here. Some are indented 4 spaces, some 6, some 8. See [our code style](https://github.com/openmrs/openmrs-contrib-android-client#Code-Style) section.

**Comment by f4ww4z** [comment]
@ribhavsharma Please also add links (as buttons) to our forums page, talk.openmrs.org and [OpenMRS IRC](https://wiki.openmrs.org/display/IRC/Home)

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java (hunk 6 above)
Comment (ribhavsharma) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java]: I get an error unless I do that :/

**Pull request closed without merging**